code/game/machinery/computer/ai_core.dm

```diff
@@ -150,7 +150,10 @@
 
 /obj/structure/AIcore/deactivated/attackby(var/obj/item/device/aicard/A as obj, var/mob/user as mob)
 	if(istype(A, /obj/item/device/aicard))//Is it?
-		A.transfer_ai("INACTIVE","AICARD",src,user)
+		if(do_after(user, src, 40))
+			if(!istype(src) && !istype(A))
```

Comment:
these two checks will never fail

Reply:
don't know if true but i'll trust you on it